src/translations/locale/locale.en-US.json

```diff
@@ -0,0 +1,4 @@
+{
+  "validation.email.invalid": "Email is not correct",
+  "validation.required": "Field is required"
+}
```

Comment:
I am using `react-intl`, basically yes, we add here all the translations and then, normally, we should use `<FormattedMessage value="validation.required" />`, but it doesn't work with the translation messages, that's why I had to create this `mapValidationMessagesToLocale` and then use `formatMessage` function that comes from the `intl` object.

The `intl` object comes from the component `context.intl` and it is added by `IntlProvider`. See in `client.tsx`:
```<Provider store={store} key="provider">
    <IntlProvider locale={language} messages={messages}>
      {component}
    </IntlProvider>
  </Provider>```